interview-questions.md

```diff
@@ -10,40 +10,79 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 Your answer:
 
+Object-oriented programming is a a computer programming design that helps us developers organize our software designs based on computer data and it's objeacts, instead of functions and logic. The major differnce between object- oriented programming and functional programming is the way each program stores and manipulates its data.  
+
 Researched answer:
 
+Research taught me that you can manipulate data simply by storing it. For example in data forensics in order to maintain the data integity you have to leave the computer on and do not save any files because even truning off the computer can manipulate data. I have never delt with the FBI before, but I watch alot of FBI shows and the FBI never wants you to touch the computer. The FBI believes computer shouldnt be touched and left as is, because once its touched data has been manipulated. As we already know once we manipulate data it losses it's intergrity.
```

Comment:
I love this answer -- however it doesn't feel related to OOP. I'd like to know what you learned about Object Oriented Programming in you research. OOP deals with encapsulation, inheritance, polymorphism, and abstraction.

 This is a great answer for data forensics.